docs/data-sources/property_rules_template.md

```diff
@@ -34,7 +34,7 @@ You can also set up a single JSON template file that calls individual template f
     ]
 ```
 
-You have to store all files in the directory listed in the `template_file` argument.
+You have to store all files in the directory listed in the `template_dir` argument.
```

Comment:
I don't think this is a valid change. The above line is part of the segment about "Use a single JSON rule tree file". In that you can't use `template_dir` argument as that is part of `template` which is the below case.

Reply:
@kaush- removed.